components/Academy/OpenToAll/OpenToAll.module.scss

```diff
@@ -0,0 +1,162 @@
+.container {
```

Comment:
In the meantime I could add extra padding to this element so it matches designs, something I noted in my MR description - I think it should scale fine depending on zoom as mobile styles will kick in to prevent this from happening too close